File: C/292.nim-game.c
/*
 * @lc app=leetcode id=292 lang=c
 *
 * [292] Nim Game
 */

// @lc code=start
#include <stdbool.h>

bool canWinNim(int n)
{
    return n % 4 != 0;
}
// @lc code=end

File: PHP/292.nim-game.php
/*
* @lc app=leetcode id=292 lang=php
*
* [292] Nim Game
*/

// @lc code=start
class Solution {

/**
* @param Integer $n
* @return Boolean
*/
function canWinNim($n) {
return $n % 4 != 0;
}
}
// @lc code=end